src/components/controls/Pagination/styles.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components'
+
+export const PaginationContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  padding-bottom: 25px;
+`
+export const PaginationList = styled.ul`
+  list-style: none;
+  padding: 0px;
+  display: flex;
+`
+
+const Li = styled.li`
+  background: #845ec2;
+  padding: 10px 15px;
+  margin-right: 4px;
+  border-radius: 4px;
+  font-family: sans-serif;
+  color: #fff;
+  font-size: 0.85rem;
+  min-width: 20px;
+  text-align: center;
+  user-select: none;
+  :hover {
+    background: #d65db1;
+  }
+`
+
+export const LiPrev = styled(Li)`
+  background: ${props => (props.disabled ? '#fbeaff' : '#3e207c')};
```

Comment:
`${props => (props.disabled ? '#fbeaff' : '#3e207c')}` to `${({ disabled })=> (disabled ? '#fbeaff' : '#3e207c')}` in all files for all props in styled components